src/NuGet.Clients/PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -196,7 +196,7 @@ private void LoadItems(PackageItemListViewModel selectedPackageItem)
             await NuGetUIThreadHelper.JoinableTaskFactory.RunAsync(async () =>
             {
                 await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
-                _loadingStatusBar.ItemsLoaded = PackageItems.Count();
+                _loadingStatusBar.ItemsLoaded = currentLoader.State.ItemsCount;
```

Comment:
Actually it cannot be null. It's a local variable with captured value of _loader at the moment search started. It correlates to the current search operation. So that's guaranteed to be not null.
While _loader is a member and always is accessed from the main UI thread. The line you referred to is a button click handler. There is a chance control state is altered after button is shown.